index.md

```diff
@@ -0,0 +1,32 @@
+# Protractor helper's library
+
+[![npm version](https://badge.fury.io/js/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper) [![npm weekly downloads](https://img.shields.io/npm/dw/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper) 
+
+This library contains helper functions that can be used together with Protractor for creating robust end-to-end tests.
+
+Many of the helper functions on this library uses `protractor.ExpectedConditions` to ensure that the elements we want to interact with are in the correct state before interacting with them or running expectations on them. This helps on avoiding trying to interact with elements when they are still not ready for it, which helps on avoiding test flakiness.
+
+## Installation
+
+Below it is described the process of Installation of such module.
+
+Run `npm install protractor-helper --save-dev` to install the library as a dev dependency of your project.
+
+## How to use and examples
+
+After installing the library you will need to require it in your test file (see below).
+
+```js
+const protractorHelper = require("protractor-helper");
+```
+
+As soon as you have the library required in your test file you can start using its helper methods.
```

Comment:
Let's change it to:
`As soon as you have the library required in your test file you can start using its helper functions.`

Reply:
Done.